apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -519,10 +557,19 @@ public String completeMultipartUpload(MultipartUpload mpu, List<MultipartPart> p
          String path = mpu.containerName() + "/" + getMPUPartName(mpu, part.partNumber());
          builder.add(Segment.builder().path(path).etag(part.partETag()).sizeBytes(part.partSize()).build());
       }
-
       return api.getStaticLargeObjectApi(regionId, mpu.containerName()).replaceManifest(mpu.blobName(),
             builder.build(), mpu.blobMetadata().getUserMetadata(), getContentMetadataForManifest(mpu.blobMetadata().getContentMetadata()));
    }
+   
+   public String completeMultipartUploadDLO(MultipartUpload mpu, List<MultipartPart> parts) {
```

Comment:
jclouds consists of two levels, the lower-level provider APIs, e.g., `DynamicLargeObjectApi`, and the higher-level portable abstraction, e.g., `RegionScopedSwiftBlobStore`.  The former exposes all the functionality of a given provider, e.g, SLO, DLO, etc., while the latter only implements a portable layer which is common between providers.  Thus `completeMultipartUploadDLO` should not exist and if callers want this functionality they can use `RegionScopedSwiftBlobStore`.